collections/_article/development-fund-perks.md

```diff
@@ -0,0 +1,64 @@
+
+Not a fund member yet? [Subscribe now!](https://fund.godotengine.org/#membership)
+
+## Connecting your accounts
+
+### Forum Badge
+
+The forum badges are assigned automatically to your Godot account when you become a Development Fund member. Make sure to use the same Godot account on the forum!
+
+![Badge with a golden Godot logo and the text "Fund Member - Member of the Development Fund", awarded to 47 users](/storage/blog/development-fund-perks/forum-badge.webp)
+
+Already a Development Fund member but your badge does not show up? Make sure that you have signed in at least once to the [development fund](https://fund.godotengine.org/#membership). It may take up to 24 hours for the badge to appear on the forum.
+
+### Discord Role
+
+Make sure you are subscribed and logged in to the [Development Fund](https://fund.godotengine.org/#membership), then follow these instructions:
+
+1. Authorize our Discord application by clicking "Connect with Discord":
+   <div class="preview-image-container">
+      <img alt='Screenshot of the Fund page showing the "Connect with Discord" option' src='/storage/blog/development-fund-perks/fund-website.webp'/>
+      <img alt='Screenshot of discord prompt asking for Authorization' src='/storage/blog/development-fund-perks/discord-auth.webp'/>
+   </div>
+2. In the options for our [Discord server](https://discord.com/invite/7HtykRQ4Rg), find and select the "Linked Roles" option
+   <div class="preview-image-container">
+      <img alt='Screenshot of discord server menu with "Linked Roles" option' src='/storage/blog/development-fund-perks/discord-menu.webp'/>
+   </div>
+3. Choose the "Fund Member" role
+   ![Screenshot of discord prompt asking for Authorization](/storage/blog/development-fund-perks/discord-select-role.webp)
+4. Click "Finish" to complete the process. Without finishing, the role won't show up!
+   ![Screenshot of discord prompt asking for Authorization](/storage/blog/development-fund-perks/discord-finish.webp)
+
+## Not ready to become a member yet?
+
+While this won't give you a fancy badge or role, we have recently added the option for one-time donations. With this update, we also switched to Stripe as our payment provider to offer the option to donate via credit card in addition to PayPal.
+For those of you that have been with the project for a long time already, you might remember our [Patreon](patreon.com/godotengine). In case you haven't already, we encourage you to move your donation over to this self-hosted solution, as we receive more of your contributions here without the provider fees. Nonetheless, the page will stay up for the time being and also connect to the Discord mentioned above.
```

Comment:
```suggestion

For those of you that have been with the project for a long time already, you might remember our [Patreon](patreon.com/godotengine). In case you haven't already, we encourage you to move your donation over to this self-hosted solution, as we receive more of your contributions here without the provider fees. Nonetheless, the page will stay up for the time being and also connect to the Discord mentioned above.
```